src/site/content/en/handbook/web-dev-components/index.md

```diff
@@ -322,7 +322,12 @@ element, and add the `w-ml--l` class to the label. See below:
 
 ## Code pattern {: #codepattern }
 
-A component that displays a demo and code snippets, organized in tabs, side by side.
+A component that displays a demo and code snippets, organized in tabs,
+side by side.
+
+The height of the component is determined by the length of the longest code
+snippet. It can be overridden by passing the optional height attribute
+in the shortcode.
```

Comment:
Height is determined by length? Is this because of wrapping?

Reply:
I think this means "code length" in the sense of, long code will cause lots of newlines.

Reply:
@jpmedley Please unblock this PR by removing "requested changes" review